src/global.js

```diff
@@ -122,19 +122,9 @@ export function getDisableSetCookie(): boolean {
   return false;
 }
 
-export function getExperimentation(): Experimentation | null {
```

Comment:
you may want to leave this function as is until you can remove it from the other components that are using it.

Unless its not being used right now

Reply:
sounds good, I don't think it is being used but I'd rather keep it than find out via deployment errors.

I'll add a `// TODO first-render-experiment-cleanup <description>` comment in all the places we should revisit